src/main/java/chatterbox/task/TaskList.java

```diff
@@ -0,0 +1,111 @@
+package chatterbox.task;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import chatterbox.ChatterboxException;
+import chatterbox.Storage;
+import chatterbox.ui.Ui;
+
+/**
+ * Handles the modification of and other operations relating to the task list.
+ */
+public class TaskList {
+    private static List<Task> tasks = new ArrayList<>();
+    private final Storage store;
```

Comment:
Perhaps considering renaming it to storage as store can be interpreted as a verb.